concepts/conditionals/about.md

```diff
@@ -1,48 +1,140 @@
-# About
+# Conditionals
 
-An `if` statement can be used to conditionally execute code:
+Ruby has what is known as control expressions, these are used to control the way the program will run and they take a truthy or falsey value.
+There are operators that can be used to create truthy or falsey values, these are known as [comparison operators][comparison-operators].
+
+There are two main control expressions that are used to control which code will run and which will not.
+Also known as which given branch will run.
+
+Those two are: `if` and the `unless` expression.
+
+## Comparison operators
+
+[Comparison operators][comparison-operators] are used to compare values and return a `true` or `false` value.
+The following operators require two values to be compared of the same type.
+If the values are not of the same type then the compiler will throw an error.
+Here is a list of the operators and an example of when they give a `true` value:
+
+| Method | Description           | Example |
+| ------ | --------------------- | ------- |
+| <      | less than             | 5 < 4   |
+| <=     | less than or equal    | 4 <= 4  |
+| >      | greater than          | 3 > 1   |
+| >=     | greater than or equal | 2 >= 2  |
+
+The equal and not equal operators can be used to compare any type of value contrary to the operators already mentioned.
+The `==` operator is used to check if two values are equal, and that includes checking the type of the value.
+The `!=` works the same way but it will return `true` if the values are not equal and `false` if they are equal.
+Here is a list of the equal and not equal operators and an example of when they give a `true` value:
+
+| Method | Description  | Example |
+| ------ | ------------ | ------- |
+| ==     | equal        | 4 == 4  |
+| !=     | not equal    | 5 != 4  |
+
+## Combined comparison operator
+
+The combined comparison operator (sometimes called spaceship operator) is a special comparison operator.
+It is special in the sense that it doesn't return a truthy or falsey value but it returns a number.
+It is written as `<=>` and it is used to compare 2 values.
+It will return `1` if the left value is greater than the right value, `-1` if the left value is less than the right value, and `0` if the values are equal.
 
 ```ruby
-x = 5
+1 <=> 2 # => -1
+2 <=> 2 # => 0
+3 <=> 2 # => 1
+```
 
-if x == 5
-  # Execute logic if x equals 5
-elsif x > 7
-  # Execute logic if x greater than 7
-else
-  # Execute logic in all other cases
+## If statement
+
+The [`if`][if] statement is used to check if a given condition is truthy or falsey.
```

Comment:
```suggestion
The [`if`][if] statement is used to check if a given condition is "truthy" or "falsey".
```

Quotes since this is not a "normal" expression, things being either truth or false, while we are introducing an idea of not strict Boolean evaluation.

Not a blocker, though, for sure.